### ISSUE TITLE
Ledger

### ISSUE THREAD

**Issue by FatBirdie** [iotaledger/firefly#1236]
Hi
Dom mentioned in a video several weeks ago, that Firefly 1.1 would have Ledger support. 1.1 has been out for a good while now, and there still isn't any Ledger support. When will this be added?

**Comment by cunum** [comment]
It's no bug they postponed it, a lot of people are waiting on this, there's a PR open for this feature: https://github.com/iotaledger/firefly/pull/114

**Comment by r-c-k** [comment]
Testing will start very soon™
